<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="birdGameButton">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>BIRD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="match3GameButton">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>MATCH THREE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="snakeGameButton">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>SNAKE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="foodCatchButton">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>FOOD CATCHER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ExitButton">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>68</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>EXIT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QProgressBar" name="BirdProgBar">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="Match3ProgBar">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="SnakeProgBar">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 150px;
  height: 50px;
  border: 3px solid #FFDAB9;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFEFD5;
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFA07A;
}

QPushButton:hover {
  background: #FFA07A;
  color: #FFEFD5;
  font-size: 1.9em;
}</string>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="FoodCatchProgBar">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="infoButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  width: 185px;
  height: 30px;
  border: 3px solid #FFEFD5;
  border-radius: 45px;
  transition: all 0.3s;
  cursor: pointer;
  background: #FFA07A; 
  font-size: 1.2em;
  font-weight: 550;
  font-family: 'Montserrat', sans-serif;
  color:#FFEFD5;
}

QPushButton:hover {
  background: #FFEFD5;
  color: #FFA07A;
  font-size: 1.9em;
}</string>
          </property>
          <property name="text">
           <string>INFO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
